frontend/tet/admin/src/components/editor/Dropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import TetPosting from 'tet/admin/types/tetposting';
+import { useFormContext, Controller } from 'react-hook-form';
+import { Select as HdsSelect } from 'hds-react';
+import Id from 'shared/types/id';
+import { RegisterOptions } from 'react-hook-form';
+
+type OptionType = {
+  value: string;
+  label: string;
+};
+
+type Props = {
```

Comment:
Dropdown is quite common component, so it would be nice to move instantly to `shared` folder so other projects could use it too. Only thing to change is the `TetPosting` to some generic like `T` then use it like `<DropDown<TetPosting> ... >`